### PR TITLE
Update docs for CI

### PR DESCRIPTION
**Pull request by wolf99**
There was a question on Slack about the CI and specifically about the decision to stick with Ubuntu 18.04.
So I have taken the opportunity to update the docs on these points as lots has changed.

The docs will probably need changing a bit for v3 anyway, but I figure this information will be the same at least.

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ See [implement-an-exercise-from-specification](https://github.com/exercism/docs/
 
 The structure of en exercise's directory is as follows (note the differing hyphen and underscore usage for directory names and file names respectively):
 
-```
-+-- <my-exercise>
+<pre>
++-- &lt;my-exercise&gt;
```
Comment (ryanplusplus) [inline docs/CONTRIBUTING.md:92]: Do you need to use `&lt;` below as well? The rich diff shows that it's messed up as-is.

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -121,17 +118,50 @@ FYI - to get the install artifact for the configlet tool, checkout the configlet
 
 ## Continuous Integration & Tests
 
-The repository uses Travis CI to run some scripts on each commit to a pull request.
-The simplest way to run these scripts on your own contribution is to open a PR on this repository with your changes, edit the PR title to add `[WIP]` then allow this repository's CI setup to take care of things for you automatically.
+This repository uses GitHub workflows to run some scripts on each commit to a pull request.
+The simplest way to run these scripts on your own contribution is to open a PR on this repository with your changes, set the PR draft, then allow this repository's CI setup to take care of things for you automatically.
```
Comment (ryanplusplus) [inline docs/CONTRIBUTING.md:122]:
```suggestion
The simplest way to run these scripts on your own contribution is to open a PR on this repository with your changes, set the PR as a draft, then allow this repository's CI setup to take care of things for you automatically.
```

**Comment by wolf99** [comment]
Already had another branch with work done to lint all the docs with some changes that fixed the angle brackets issue.
So I've just merged and pushed that here. It's a bit more to review though! 

Next step may be to add markdown lint CI... 
(a la https://github.com/exercism/website-copy/pull/1903)

File: docs/C_STYLE_GUIDE.md [hunk 3]
```diff
@@ -54,22 +55,24 @@ While aware that POSIX reserves the `_t` suffix, the track does not currently ut
...
+The repository uses the `indent` tool, as outlined in the [contributing guide][].
 
-The options described for use with indent there are `-linux -i3 -nut`. The `-linux` option is a shortcut that is equivelent to a secific fixed group of options. The combined equivalent options are `-nbad -bap -nbc -bbo -hnl -br -brs -c33 -cd33 -ncdb -ce -ci4 -cli0 -d0 -di1 -nfc1 -i3 -nut -ip0 -l80 -lp -npcs -nprs -npsl -sai -saf -saw -ncs -nsc -sob -nfca -cp33 -ss -il1`.
+The options described for use with indent there are `-linux -i3 -nut`. The `-linux` option is a shortcut that is equivalent to a specific fixed group of options. The combined equivalent options are `-nbad -bap -nbc -bbo -hnl -br -brs -c33 -cd33 -ncdb -ce -ci4 -cli0 -d0 -di1 -nfc1 -i3 -nut -ip0 -l80 -lp -npcs -nprs -npsl -sai -saf -saw -ncs -nsc -sob -nfca -cp33 -ss -il1`.
 
-What these options do is indicated by the [GNU `indent` manual](https://www.gnu.org/software/indent/manual/indent.html#SEC4), but is also described here for easy reference if editing files manually.
+What these options do is indicated by the [GNU `indent` manual][ident-manual], but is also described here for easy reference if editing files manually.
```
Comment (ryanplusplus) [inline docs/C_STYLE_GUIDE.md:75]:
```suggestion
What these options do is indicated by the [GNU `indent` manual][indent-manual], but is also described here for easy reference if editing files manually.
```

File: docs/C_STYLE_GUIDE.md [hunk 4]
```diff
@@ -177,3 +181,7 @@ int main(void)
     return 0;
 }
 ```
+
+[Linux kernel coding style guide]: https://www.kernel.org/doc/html/latest/process/coding-style.html
+[contributing guide]: ./CONTRIBUTING.md
+[ident-manual]: https://www.gnu.org/software/indent/manual/indent.html#SEC4
```
Comment (ryanplusplus) [inline docs/C_STYLE_GUIDE.md:187]:
```suggestion
[indent-manual]: https://www.gnu.org/software/indent/manual/indent.html#SEC4
```

**Review by ryanplusplus — APPROVED** [review]

**Pull request merged**